.buildkite/pipeline.test.yml

```diff
@@ -18,6 +18,11 @@
       --test_tag_filters=release_unit
       release/...
 
+- label: ":shield: release tests"
```

Comment:
Let's use a more descriptive name here. "Release tests" suggests that we run release tests in here, which we don't :-)

Reply:
Totally!